Extensions/PathfindingBehavior/tests/legacypathfindingruntimebehavior.spec.js

```diff
@@ -770,7 +692,7 @@ describe('gdjs.PathfindingRuntimeBehavior', function () {
           describe(`centered: ${objectCenteredOnCells},`, function () {
             ['left', 'right', 'down', 'up'].forEach((direction) => {
               describe(`direction: ${direction})`, function () {
-                legacypathfindingtest.doTests(
+                doLegacyPathFindingTests(
```

Comment:
Same thing here, can we:
- do these changes in a separate PR, which is only about tests? :) 
- Move `doLegacyPathFindingTests ` inside the `describe` to avoid global variables ;)

Reply:
Ok, the changes on the tests will be merged before or after this one?

Reply:
Before :) 
Basically the idea is to set up in a separate PR as much tests as possible, so that: 
- the existing behavior is very well tested
- this PR size decrease (because there is little change in the test compared to the one merged previously) 

and this will reduce the chance of myself missing bugs/mistakes during the review (because if I review both tests that were changed *and* the behavior that was changed, I can miss mistakes in both).

I know this is a bit of a pain, but this will be very useful. Basically a PR containing only tests, I can merge it very quickly :)

Reply:
No, it's ok, I'm becoming a rebase expert :sunglasses:

Reply:
Hehe! :) That's where git shines (and actually Mercurial might even be easier to use, you can go back and rebase your commits very easily).